### PR TITLE
Release 2.0

### PR DESCRIPTION
**Pull request by mzbroch**
# ntc-netbox-plugin-onboarding v2.0 Release Notes

### Enhancements

* NetBox 2.9 support - Supported releases 2.8 and 2.9
* Onboarding extensions - Customizable onboarding process through Python modules.
* Onboarding details exposed in a device view - Date, Status, Last success and Latest task id related to the onboarded device are presented under the device view.
* Onboarding task view - Onboarding details exposed in a dedicated view, including NetBox's ChangeLog.
* Onboarding Changelog - Onboarding uses NetBox's ChangeLog to display user and changes made to the Onboarding Task object.
* Skip onboarding feature - New attribute in the OnboardingDevice model allows to skip the onboarding request on devices with disabled onboarding setting.

### Bug Fixes

* Fixed race condition in `worker.py`
* Improved logging

### Additional Changes

* Platform map now includes NAPALM drivers as defined in NetBox
* Tests have been refactored to inherit NetBox's tests
* Onboarding process will update the Device found by the IP-address lookup. In case of no existing device with onboarded IP-address is found in NetBox, onboarding might update the existing NetBox' looking up by network device's hostname.
* Onboarding will raise Exception when `create_device_type_if_missing` is set to `False` for existing Device with DeviceType mismatch (behaviour pre https://github.com/networktocode/ntc-netbox-plugin-onboarding/issues/74)
* Extended search criteria

### CODE REVIEW

File: netbox_onboarding/__init__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 limitations under the License.
 """
 
-__version__ = "1.3.0"
+__version__ = "2.0.0-beta.2"
```
Comment (dgarros) [inline netbox_onboarding/__init__.py:15]: Please can you update the version

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ntc-netbox-plugin-onboarding"
-version = "1.3.0"
+version = "2.0.0-beta.2"
```
Comment (dgarros) [inline pyproject.toml:3]: Please update the version here as well

**Review by dgarros — APPROVED** [review]
Exciting ..
LGTM minus the version that needs to be updated in 2 places

File: netbox_onboarding/__init__.py (hunk 1 above)
Comment (mzbroch) [reply netbox_onboarding/__init__.py]: Version updated

File: pyproject.toml (hunk 2 above)
Comment (mzbroch) [reply pyproject.toml]: Version updated

**Pull request merged**